tests/data/car.py

```diff
@@ -24,15 +24,15 @@
         (_neat["Blue"], RDF.type, _neat["Color"]),
         (_neat["Blue"], _neat["name"], Literal("blue")),
         (_neat["Car1"], RDF.type, _neat["Car"]),
-        (_neat["Car1"], _neat["Car.Manufacturer"], _neat["Toyota"]),
+        (_neat["Car1"], _neat["make"], _neat["Toyota"]),
         (_neat["Car1"], _neat["year"], Literal(2020)),
         (_neat["Car1"], _neat["color"], _neat["Blue"]),
         (_neat["Ford"], RDF.type, _neat["Manufacturer"]),
         (_neat["Ford"], _neat["name"], Literal("Ford")),
         (_neat["Red"], RDF.type, _neat["Color"]),
         (_neat["Red"], _neat["name"], Literal("red")),
         (_neat["Car2"], RDF.type, _neat["Car"]),
-        (_neat["Car2"], _neat["Car.Manufacturer"], _neat["Ford"]),
```

Comment:
this was wrong, as actual property that holds edges is `make`, `Car.Manufacturer` is edge type

Reply:
Probably explains how the bug was introduced in the first place.